src/BridgeIHMCORS.h

```diff
@@ -31,6 +32,38 @@
 namespace yarp {
 namespace dev {
 
+class BridgeIHMCORS;
+
+class BridgeIHMCORSReceiverThread : public yarp::os::Thread
+{
+    BridgeIHMCORS * m_parentDevice;
```

Comment:
Given that the thread cannot exist without the parent device, I would make this explicit and use a reference.
Then, move the initialisation of the reference in the constructor instead of the configure

Reply:
I tend to avoid doing that because then you need to have a new/delete pair for allocating/deallocating the `BridgeIHMCORSReceiverThread`  in the device, but I don't see any big difference between the two approaches, so I can switch it to a reference.